config/ember-try.js

```diff
@@ -17,72 +19,6 @@ module.exports = {
       }
     },
     {
```

Comment:
Restore this, or use the new style config: https://github.com/ember-cli/ember-try#config